packit_service/service/views.py

```diff
@@ -72,35 +65,23 @@ def _get_build_logs_for_build(
     else:
         title_identifier = ""
 
-    response = (
-        "<html><head>"
-        f"<title>{build_description} {project.namespace}/{project.repo_name}:"
-        f" {title_identifier}</title></head><body>"
-        f"{build_description} ID: {build.build_id}<br>"
-        f"Submitted: {models.optional_time(build.build_submitted_time)}<br>"
-        f"State: {build.status}<br><br>"
-        f'Build web interface URL: <a href="{build.web_url}">{build.web_url}</a><br>'
+    return render_template(
+        "build_logs.html",
+        project=project,
+        title_identifier=title_identifier,
+        build_description=build_description,
+        build=build,
+        build_submitted_time=models.optional_time(build.build_submitted_time),
```

Comment:
I am missing this info in the templates.

Reply:
Oh, yes, I forgot about this...

The reason it's not in the template is b/c I couldn't come up with a good reason why it would be useful for the users to be there. The information is already on the Copr-build page we link, and that page gives more context on the build. So I just didn't see the point in a lonely build submitted timestamp on our page :)

My idea is to keep our log pages as minimal, clean and low-maintenance as possible.

Should we keep this timestamp? If no, I'll clean up here, if yes, I'll add to the template.

Reply:
- I -- as a user -- would not like to click multiple times to get that basic information.
  - And in copr, you don't have the reverse link to get to our build page.
- This page should be accessible in long-term -- github-status can change and copr as well. (Outage, removed temporary project,...)
- You can easily compare multiple builds for one PR.
- Having basic information can help us solve potential problems more easily.
  - Users can send us this link => The content is stable and we can read everything needed. (That relates to the second one: PR can change, copr can change.)
  - That's why I added that.
- I would try to use more styling instead of data cleanup to make the page cleaner.